examples/line-liff-v1/server.js

```diff
@@ -0,0 +1,50 @@
+const path = require('path');
+
+const bodyParser = require('body-parser');
+const express = require('express');
+const { bottender } = require('bottender');
+const ejs = require('ejs');
+
+const app = bottender({
+  dev: process.env.NODE_ENV !== 'production',
+});
+
+const port = Number(process.env.PORT) || 5000;
+
+const handle = app.getRequestHandler();
+
+app.prepare().then(() => {
+  const server = express();
+
+  server.use(
+    bodyParser.json({
+      verify: (req, _, buf) => {
+        req.rawBody = buf.toString();
+      },
+    })
+  );
+
+  server.get('/api', (req, res) => {
```

Comment:
We should remove those /api stuff, because it only be used to demo how to add APIs to your custom server in custom server example, but it's not quite useful in this example.